### DISCUSSION TITLE
本地调试

### DISCUSSION THREAD

**Discussion by Burninghamster** [illacloud/illa-builder#722] [Q&A]
本地命令行安装部署那个我没啥问题，但是我想在本地做本地调试，虽然你们不提倡说麻烦，但是我还是想在本地调试。项目依赖我下载不下来 （本地调试步骤是啥？？？）或者官方可不可以把你们本地下载好的依赖已邮件的形式压缩发给我也行，illa-design这个依赖我可以正常下载，然后后面怎么运行，怎么跑我就研究不出来了。

**Comment by AruSeito** [comment]
您好，请问您是指哪个项目依赖下载不下来呢？目前还没有对本地开发流程进行优化，强烈不建议在当前阶段进行本地开发。
如果您需要本地开发可以参考如下步骤：
```
git clone git@github.com:illacloud/illa-builder.git
cd illa-builder
git submodule init && git submodule update
pnpm install
pnpm dev
```
如果需要启动 dev，需要在`apps/builder/.env.cloud.local`配置
```
VITE_API_BASE_URL=localhost:3000/api/v1   // 端口号按照您本地浏览器输入的地址来输入即可。
VITE_PROXY_API_BASE_URL=http(s)://localhost:9999  // 这个按照您本地起的后端服务来输入。
```

然后您直接在 浏览器输入 `https://localhost:3000` 即可。

如果您在后续中有什么问题，欢迎您在这里继续回复讨论。

**Reply by Burninghamster** [reply]
<img width="854" alt="7444154d108d6863611f815dc006ebc" src="https://user-images.githubusercontent.com/33534997/209926302-bebb0bf6-4285-45e6-b34d-5663bdf24b78.png">
https://registry.npm.taobao.org/@uiw/react-color-material/-/react-color-material-1.1.1.tgz这个依赖下载不下来

**Comment by Burninghamster** [comment]
这个我已经解决了，不用看了，就是不知道打包命令是什么
